_redirects

```diff
@@ -270,6 +270,24 @@
 /archive/v1.5/getting-started/kubernetes/cloud-config/*          https://calico-legacy.netlify.app/archive/v1.5/getting-started/kubernetes/cloud-config/:splat      200
 /archive/v1.5/getting-started/kubernetes/tutorials/*             https://calico-legacy.netlify.app/archive/v1.5/getting-started/kubernetes/tutorials/:splat         200
+                                
+# Some specific pages are throwing 404 because of the docs restructuring
+/master/getting-started/kubernetes/minikube                      https://docs.tigera.io/calico/next/getting-started/kubernetes/minikube                                      301!
+/networking/use-specific-ip                                      https://docs.tigera.io/calico/latest/networking/ipam/use-specific-ip                                        301!
+/master/getting-started/kubernetes/flannel/flannel               https://docs.tigera.io/calico/next/getting-started/kubernetes/flannel/                                      301!
+/networking/get-started-ip-addresses                             https://docs.tigera.io/calico/latest/networking/ipam/get-started-ip-addresses                               301!
+/master/reference/private-cloud/l3-interconnect-fabric           https://docs.tigera.io/calico/next/reference/architecture/design/l3-interconnect-fabric                     301!
+/security/services-cluster-ips                                   https://docs.tigera.io/calico/latest/network-policy/services/services-cluster-ips                           301!
+/security/kubernetes-network-policy                              https://docs.tigera.io/calico/latest/network-policy/get-started/kubernetes-policy/kubernetes-network-policy 301!
+/networking/mtu                                                  https://docs.tigera.io/calico/latest/networking/configuring/mtu                                             301!
+/maintenance/ebpf/use-cases-ebpf                                 https://docs.tigera.io/calico/latest/operations/ebpf/use-cases-ebpf                                         301!
+/reference/contribute                                            https://docs.tigera.io/calico/latest/reference/involved                                                     301!
+/security/calico-network-policy                                  https://docs.tigera.io/calico/latest/network-policy/get-started/calico-policy/calico-network-policy         301!
+/maintenance/troubleshoot/vpp                                    https://docs.tigera.io/calico/latest/operations/troubleshoot/vpp                                            301!
+/getting-started/docker/tutorials/docker-swarm                   https://docs.tigera.io/calico/latest/getting-started                                                        301!
+/master/maintenance/monitor/monitor-component-visual             https://docs.tigera.io/calico/next/operations/monitor/monitor-component-visual                              301!
```

Comment:
The list from our test included this same article for both latest and master. Can you add the same rule for latest? but without `/master`. Should be:
`/maintenance/monitor/monitor-component-visual https://docs.tigera.io/calico/latest/operations/monitor/monitor-component-visual 301`